### PR TITLE
[fr] Follow #5186, #5191, #5190, #5192, #5196, #5197, #5198

### PR DESCRIPTION
**Pull request by antograssiot**

### CODE REVIEW

File: fr/appendices/3-5-migration-guide.rst [hunk 1]
```diff
@@ -269,6 +271,13 @@ Http
 * ``Cake\Http\Client::addCookie()`` a été ajoutée pour faciliter l'ajoute de cookies
   à une instance d'un client.
 
+InstanceConfigTrait
+-------------------
+
+* ``InstanceConfigTrait::getConfig()`` accepte maintenant un second paramèrte
```
Comment (cake17) [inline fr/appendices/3-5-migration-guide.rst]: paramètre

File: fr/appendices/3-5-migration-guide.rst [hunk 2]
```diff
@@ -269,6 +271,13 @@ Http
...
+  ``$default``. Si aucun valeur n'est disponible pour la ``$key`` spécifiée, la
```
Comment (cake17) [inline fr/appendices/3-5-migration-guide.rst]: Si aucune

File: fr/console-and-shells.rst [hunk 3]
```diff
@@ -1316,7 +1316,8 @@ lequel vous souhaitez envoyer l'email en faisant::
     use Cake\Mailer\Email;
 
     $email = new Email();
-    $email->domain('www.example.org');
+    // Avant 3.4 utiliez domain()
```
Comment (cake17) [inline fr/console-and-shells.rst]: utilisez

**Comment by cake17** [comment]
Only small typo 👍

**Comment by antograssiot** [comment]
@cake17 thanks for reviewing, I updated and added translation for #5196, #5197, #5198

**Comment by HavokInspiration** [comment]
Good job @antograssiot !

**Pull request merged**